src/granite/base.cr

```diff
@@ -43,7 +43,7 @@ class Granite::Base
     set_attributes(args.to_h)
   end
 
-  def initialize(args : Hash(Symbol | String, String | JSON::Any))
+  def initialize(args : Hash(Symbol | String, JSON::Any::Type))
```

Comment:
Lets define a initialize overload, I think it would be better
```crystal
def initialize(args : Hash(Symbol | String, JSON::Any::Type))
  new(JSON.parse(args))
end

def initialize(args : Hash(Symbol | String, JSON::Any))
end
```